manual/python/index.md

```diff
@@ -7,13 +7,14 @@ toc: true
 toc_sticky: true
 ---
 
-ROOT can be used from Python thanks to its Python-C++ bindings, called PyROOT. PyROOT is HEP's entrance to all C++ from Python, for example, for frameworks and their steering code. The PyROOT bindings are *automatic* and *dynamic*: no pre-generation of Python wrappers is necessary.
+With PyROOT you can use ROOT, because of its Python-C++ bindings, from Python. PyROOT is HEP's entrance to all C++ from Python, for example, for frameworks and their steering code. The PyROOT bindings are *automatic* and *dynamic*: no pre-generation of Python wrappers is necessary.
```

Comment:
Here I would like to make it clear that PyROOT is the Python bindings, so perhaps something like `With PyROOT, ROOT's Python-C++ bindings, you can use ROOT from Python`?